packages/gatsby-theme-carbon/src/components/Tabs/Tabs.module.scss

```diff
@@ -1,106 +1,57 @@
-div.tab-container {
-  margin-left: -$spacing-05;
-  min-height: 2.5rem;
-  width: 100vw;
-  @include carbon--breakpoint('md') {
-    width: 100%;
-  }
+.tab-list {
+  display: flex;
+  margin-left: -1rem;
+  margin-right: -1rem;
+  scrollbar-width: none;
+}
+
+.tab-list li:focus-within {
+  outline: 2px solid blue;
```

Comment:
Would we want to use our $focus color token here? Focus states look like they need a design spec also

Reply:
@jeanservaas is takin a look at if we need a more formal spec for some aspects